lib/rules/max-nesting-depth/README.md

```diff
@@ -148,6 +148,65 @@ a {
 }
 ```
 
+### `ignore: ["blockless-pseudoclasses"]`
+
+Ignore pseudoclases that only wrap other rules, and do not themselves have declaration blocks.
+
+For example, with `1`:
+
+The following patterns are considered violations:
+
+As the pseudoclasses have a declarations blocks.
+
+```css
```

Comment:
I'm not sure this is a violation:

```css
a {
  &:hover { /* ignored */
    b { /* 1 */
      top: 0;
    }
  }
}
```

Reply:
You are right, I update that and also the examples given to be easier to read